### PR TITLE
CRI-O: new e2e features jobs

### PR DESCRIPTION
**Pull request by runcom**
@mrunalp @stevekuznetsov ptal

Signed-off-by: Antonio Murdaca <runcom@linux.com>

### CODE REVIEW

File: sjb/config/test_cases/test_pull_request_crio_e2e_features_rhel.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      cd /go/src/github.com/kubernetes-sigs/cri-o
+      git fetch origin
+      git checkout master
+      git branch -D target || true
+      git branch -D pr || true
+      git branch target "${PULL_BASE_SHA}"
+      git checkout target
+      git fetch origin "pull/${PULL_NUMBER}/head:pr"
+      git merge "${PULL_PULL_SHA}"
+  - type: "script"
```
Comment (runcom) [inline sjb/config/test_cases/test_pull_request_crio_e2e_features_rhel.yml]: @stevekuznetsov I'm wondering if it possible to override the actions above so I can pick them up directly from the base e2e script and just override this script for the new jobs

Comment (stevekuznetsov) [reply sjb/config/test_cases/test_pull_request_crio_e2e_features_rhel.yml]: you could make a base job that only does the clone, but i don't think it's worth it -- that code has never changed, right?

Comment (runcom) [reply sjb/config/test_cases/test_pull_request_crio_e2e_features_rhel.yml]: yeah it never changed indeed, was just wondering though :)

**Comment by stevekuznetsov** [comment]
Is this good to go?

**Comment by runcom** [comment]
Yeah afaict

**Comment by runcom** [comment]
@stevekuznetsov ptal this is ready to merge along with https://github.com/openshift/release/pull/2101 and https://github.com/openshift/test-infra/pull/178

**Pull request merged**